### ISSUE TITLE
Test SSH before trying to connect

### ISSUE THREAD

**Issue by AstrorEnales** [BiBiServ/bibigrid#57]
Instead of looping through the ssh and sftp connect process, poll if ssh is available

**Comment by jkrue** [comment]
```
function check_service {
  /bin/nc ${1} ${2} </dev/null 2>/dev/null
  while test $? -eq 1; do
    log "wait 10s for service available at ${1}:${2}"
    sleep 10
    /bin/nc ${1} ${2} </dev/null 2>/dev/null
  done
}
```

**Comment by AstrorEnales** [comment]
Implemented with commit 441e6b30377b27a932f6b1bc10af50346f8e0201
SSH port is polled until available and then the normal connection is established. This probably makes the retries afterwards obsolete (now always connection on first try obviously).
